src/styles/prizes.scss

```diff
@@ -0,0 +1,39 @@
+.prize-container {
```

Comment:
I notice there is no mobile styling here. I can make it a little more responsive today.